xpano/gui/panels/about.cc

```diff
@@ -51,9 +57,11 @@ utils::Text DefaultNotice() {
 }
 }  // namespace
 
-AboutPane::AboutPane(std::vector<utils::Text> licenses)
+AboutPane::AboutPane(std::future<std::vector<utils::Text>> licenses)
     : licenses_(std::move(licenses)) {
-  licenses_.insert(licenses_.begin(), DefaultNotice());
+  // licenses_.insert(licenses_.begin(), DefaultNotice());
+  license_text.push_back({});
+  license_text.push_back(DefaultNotice());
```

Comment:
Please move this to the _member initializer list_ like this;
`: licenses_{DefaultNotice()}`